vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2163,6 +2169,16 @@ private void launchInvitePeople() {
         }
     }
 
+    /**
+     * Launch audio recorder intent
+     */
+    @SuppressLint("NewApi")
+    private void launchAudioRecorderIntent() {
+        enableActionBarHeader(HIDE_ACTION_BAR_HEADER);
+        Intent fileIntent = new Intent( MediaStore.Audio.Media.RECORD_SOUND_ACTION );
+        startActivityForResult(fileIntent, REQUEST_FILES_REQUEST_CODE);
```

Comment:
RECORD_SOUND_ACTION seems optional (https://stackoverflow.com/questions/15546051/android-mediastore-audio-media-record-sound-action-no-activity-found-to-handle).
Notice, we might have the same issue for the other entry (camera...)